docs/analysis_plan.md

```diff
@@ -0,0 +1,60 @@
+
+We will address this in one of two ways:
+1. By correcting the data for right-truncation prior to passing it to the model (pipeline approach, generally bad)
+2. Separately estimating a delay distribution for each region, and modifying the `wwinference` model to account for right-truncated data (still sort of pipelining approach, but slightly better)
+
+## Wastewater concentration data
+Data on wastewater concentrations of SARS-CoV-2 are available from individual treatment plants, indexed by sample collection date.
+Each wastewater treatment plant is contained within a state, and contains the metadata on the catchment area (population size) served by the wastewater treatment plant.
+We will assume all individuals served by the wastewater treatment plant reside within the state.
+We will jointly fit to all wastewater treatment plants within a state.
+
+# Models
+- `wwinference` (ww + hosp): This model will jointly fit to the daily hospital admissions at the state-level and the wastewater concentration data from the treatment plants within the state.
+- `wwinference` (hosp only): This model will be fit only to the daily hospital admissions at the state-level
+- baseline timeseries model (hosp only): This model will fit only to the daily hospital admissions at the state-level, using a time-series approach to generate forecasts
+
+# Forecasts
+Forecasts will be generated from each of the 3 models for all location-forecast dates where wastewater data is available.
+Forecasts will be generated at a daily temporal resoltuion, weekly, for 28 days ahead of the final forecast dates, from October 2024 to March 2025.
+
+Visual comparisons of forecasts overlaid with evaluation data will be used to assess the forecast performance of each model.
+
+## Evaluation
+The forecast performance will be evaluated against the final hospital admissions dataset available in June of 2025.
+Posterior samples of forecasts will be log-transformed, along with the observations the models are evaluated against.
+Forecasts will be scored using proper scoring rules, with metrics including the CRPS, coverage, and bias.
+
+We will summarise and analyse scores at the level of each individual forecast problem (a specific forecast date-location combination).
+This means we will summarise across the 28 horizon days to get an absolute score for each forecast.
+
+
+We will then present the results by analysing the distributions of absolute scores for each forecast in the following analyses (to be made into figures):
+- CRPS by model summarised across locations and forecast dates, broken down by under/over prediction and dispersion, in the form of a horizontal bar chart for each of the models
+- CRPS by model summarised across forecast dates, stratified by location (heatmap x-axis = model, y-axis = location)
+- CRPS by model summarised across locations, stratified by forecast date (line plot x-axis = date, y-axis = CRPS, color = model)
```

Comment:
Is a map of performance useful?

Reply:
a physical map you mean?

Reply:
yes

Reply:
potentially including geolocation of sample sites if its a thing we can get. Its mostly just to ease people in but might be interesting. 

I don't have a good understanding of the structure of german administrative districts so its going to be hard to get a good sense of any spatial relationships at least for me

Reply:
Me neither -- I don't think there's any spatial data in the raw data but can take a look

Reply:
Shall we make this an issue?